docs/data_types/image/image.md

```diff
@@ -0,0 +1,156 @@
+# 🖼️ Image
```

Comment:
Maybe somewhere we can show how you can specify the shape of the image in the types? `ImageTorchTensor[3, 224, 224]`. I think it is a cool feature!